src/splitpane/tests/functional/SplitPane.ts

```diff
@@ -7,6 +7,7 @@ import Test from 'intern/lib/Test';
 import * as css from '../../../theme/splitpane/splitPane.m.css';
 
 const DELAY = 300;
+const ERROR_MARGIN = 5;
```

Comment:
Glad this has been moved to a const, but why has it been changed from `1` to `5` within this issue?

Reply:
Just so the functional tests would actually pass (at least more frequently) :P